### ISSUE TITLE
Error when running `bspwm_swallow`

### ISSUE THREAD

**Issue by ammgws** [WnndGws/scripts#31]
```sh
git clone https://github.com/WnndGws/scripts && cd scripts/python
chmod +x bspwm_swallow
./bspwm_swallow
```

Error:
>      print(f'xprop -id {wid}')\
>                             ^
> SyntaxError: invalid syntax

README says "All python scripts are written for python3 only" and that is what I am running it with.

**Comment by WnndGws** [comment]
What is the output of `python --version`?

Likely addressed by 68a16cd

**Comment by ammgws** [comment]
3.5
